docs/index.rst

```diff
@@ -6,23 +6,44 @@
 Welcome to microraiden's documentation!
 =======================================
 
+What is µRaiden?
+-----------------
+µRaiden (read: Micro Raiden) is a payment channel framework for frequent, fast and free ERC20 token based micropayments between two parties. 
+It comes as a set of open source libraries, documentation, and code examples for multiple use cases, ready to implement on the Ethereum mainnet. 
+Whereas its big brother, the Raiden Network, aims to allow for multihop transfers via a network of bidirectional payment channels, µRaiden already enables to make micropayments through unidirectional payment channels.
 
-Contents
-=========
+.. figure:: /diagrams/uRaidenOverview.png
+   :alt:
+
+Try out the demos
+------------------
+
+We have deployed some demo applications that make extensive use of µRaiden in your browser.
+Although you need some testnet-Ether and MetaMask, it's a very easy starting point 
+to try out µRaiden.
+Just follow the instructions on  
+https://demo.micro.raiden.network/
+
+Next steps
+-----------
+If you want to start playing with microraiden, a good starting point is to check out the :doc:`Tutorials <tutorials/index>` section.
+Best you start with the :doc:`developer setup <tutorials/index>` and try our :doc:`Blockchain Tutorial <tutorials/index>` afterwards.
+
+Documentation Content
+----------------------
 
 .. toctree::
    :maxdepth: 2 
-   :caption: Contents:
 
-   proxy-tutorial
-   rest-api
-   dev-overview
-   jsclient/index
+   introduction/index
+   specifications/index
+   applications/index
    contract/index 
```

Comment:
Because I cannot comment on the diagram, this should be removed: `docs/contract/diagrams/ContractClass.png`